src/components/Checkout/StripeCheckout/StripeForm.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { FC, useMemo, useState } from 'react';
+import {
+	CardCvcElement,
+	CardExpiryElement,
+	CardNumberElement,
+} from '@stripe/react-stripe-js';
+import { Stripe, StripeElements, StripeError } from '@stripe/stripe-js';
+import { toast } from 'sonner';
+
+import Checkbox from '@/components/Onboarding/Checkbox';
+import clsxm from '@/helpers/clsxm';
+
+type StripeFormProps = {
```

Comment:
move to the interface file

Reply:
These are component props, they can stay here